files/ja/web/css/blend-mode/index.html

```diff
@@ -4,33 +4,32 @@
 tags:
   - Blend modes
   - CSS
-  - CSS Compositing
   - CSS Data Type
-  - CSS データ型
-  - CSS 合成
   - Compositing
+  - Compositing and Blending
+  - Data Type
   - Reference
+  - color
   - ブレンドモード
   - 合成
+browser-compat: css.types.blend-mode
```

Comment:
As of https://github.com/mdn/yari/pull/3955 you won't be needing to do this. In fact, I would discourage it. 
What you can do is omit this line and `<p>{{Compat}}</p>` will just work.

Reply:
Thanks, but I can't omit this line because the {{Compat}} macro is reported as a flaw if the line is omitted.
Please modify the English version first. After it has been accepted, we will adjust as so.

Reply:
That must mean your `yari` is out-of-date. I think. 
And the latest en-US content has it in there: https://github.com/mdn/content/blob/e60dc3616e284c9b36b17a4c12e694984e3ebfde/files/en-us/web/css/blend-mode/index.html#L13

Would you mind trying to merge in the latest upstream `main` into this branch and see if it stops being a flaw?